CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - PR #4386 Update Java package to 0.14
 - PR #4402 Fix cudf::strings::join_strings logic with all-null strings and null narep
 - PR #4570 Fixing loc ordering issue in dataframe
+- PR #4617 Make aggregation destructor virtual
```

Comment:
Should this instead be something like "Fix a memory leak in ..." ?

Reply:
You're right. We generally write the result of the fix rather than how we fixed something.